packages/bits/src/lib/checkbox/checkbox.component.less

```diff
@@ -50,7 +50,8 @@
         }
 
         &:focus {
-            .focus-outline-inset();
+            outline: none;
```

Comment:
I loaded this up into the browser and it looks like chrome doesn't like this value for outline. As a result when you focus one checkbox with the mouse and then hit TAB, the black outline shows up. The fix is to change `none` to `0`.

```suggestion
            outline: 0;
```

Reply:
I'm pretty sure `none` is a valid value for the `outline`.
https://www.w3schools.com/css/css_outline.asp

![image](https://user-images.githubusercontent.com/22597043/122032732-9583b980-cdd8-11eb-9581-ddad25441752.png)
![image](https://user-images.githubusercontent.com/22597043/122032831-a9c7b680-cdd8-11eb-8d5e-afa8e1a6ffc7.png)

Reply:
Okay, thanks for checking. I'm not sure what I was seeing then.